spec/controllers/api/v1/books_controller_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+describe Api::V1::BooksController do
+  context 'When user is autenticated' do
+    include_context 'Authenticated User'
+
+    describe 'GET #index' do
+      context 'When fetching all the books' do
+        let!(:books) { create_list(:book, 3) }
+
+        before do
+          get :index
+        end
+
+        it 'responses with the books json' do
+          expect(JSON.parse(response.body)).to be_paginated
+        end
+
+        it 'responds with 200 status' do
+          expect(response).to have_http_status(:ok)
+        end
+      end
+    end
+
+    describe 'GET #show' do
+      context 'When fetching a book' do
+        let!(:book) { create(:book) }
+
+        before do
+          get :show, params: { id: book.id }
+        end
+
+        it 'responses with the book json' do
+          expect(response.body).to eq ShowBookSerializer.new(
+            book, root: false
+          ).to_json
+        end
+
+        it 'responds with 200 status' do
+          expect(response).to have_http_status(:ok)
+        end
+
+        it 'responds with 404 status' do
```

Comment:
Siento esto un poco fuera de lugar, en particular porque haces un "before" con un "get" que estás pisando en este mismo "it". Por ahí poner una agrupación para ese before se vería mejor. No es nada alarmante, es un detalle más que menor!